src/state_machine_tests.zig

```diff
@@ -1202,6 +1194,7 @@ test "create_transfers/lookup_transfers" {
         // `credit_account_not_found` is a transient error, T2 cannot be reused:
         \\ transfer   T2 A1 A9    9   _  _  _  _    _ L9 C1   _ PEN   _   _   _   _ _  _   _   _ _ credit_account_not_found
         \\ transfer   T2 A1 A3  123   _  _  _  _    _ L1 C1   _ _     _   _   _   _ _  _   _   _ _ id_already_failed
+        \\ commit create_transfers
```

Comment:
Potentially stupid question, but why we need these test changes?

Reply:
It's a minor drawback: now we use the reduced `constants.message_size_max` from the test config, which allows at most 30 events.

We can either commit more often during unit tests or increase `constants.message_size_max` for tests.